notebooks/Quickstart_simple.py

```diff
@@ -0,0 +1,81 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.2
+# ---
+
+# # Quickstart - Simple avatarization
+
+# +
+# This is the client that you'll be using for all of your requests
+from avatars.client import ApiClient
+from avatars.models import AvatarizationJobCreate, AvatarizationParameters, JobStatus
+from avatars.models import ReportCreate
+from avatars.models import PrivacyMetricsJobCreate, PrivacyMetricsParameters
+from avatars.models import SignalMetricsJobCreate, SignalMetricsParameters
+
+# The following are not necessary to run avatar but are used in this tutorial
+import pandas as pd
+import io
+import os
+import datetime
+
+# -
+
+url = os.environ.get("AVATAR_BASE_URL")
+username = os.environ.get("AVATAR_USERNAME")
+password = os.environ.get("AVATAR_PASSWORD")
+print(url)
+
+# Change this to your actual server endpoint, e.g. base_url="https://avatar.company.com"
+client = ApiClient(base_url=url)
+client.authenticate(username=username, password=password)
+
+# ## Loading data
+
+df = pd.read_csv("../fixtures/iris.csv")
+dataset = client.pandas_integration.upload_dataframe(df)
+
+# ## Analyze your data
+
+while dataset.summary is None:
+    dataset = client.datasets.analyze_dataset(dataset.id)
+print(f"Lines: {dataset.nb_lines}, dimensions: {dataset.nb_dimensions}")
```

Comment:
good addition, can you add this as a section in https://python.docs.octopize.io/latest/user_guide.html ?

Reply:
their aim is to be very straightforward in order to be shown during presales demo.